composer.json

```diff
@@ -42,7 +42,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^5.7.14|^6.0",
-        "cakephp/cakephp-codesniffer": "^3.0"
+        "cakephp/cakephp-codesniffer": "^3.0",
+        "vimeo/psalm": "^1.0"
```

Comment:
Since `psalm` is being added as dependency in `travis.yml` itself, adding is here is not necessary. Most contributors aren't going to run it locally anyway.

Reply:
Absolutely. Will remove